### PR TITLE
Add EncodeConcatenatedHashes

### PR DESCRIPTION
**Pull request by chappjc**
For https://github.com/decred/dcrrpcclient/pull/39

### CODE REVIEW

File: dcrjson/parse.go [hunk 1]
```diff
@@ -87,6 +88,15 @@ func EncodeConcatenatedVoteBits(voteBitsSlice []stake.VoteBits) (string, error)
 	return hex.EncodeToString(vbBytes), nil
 }
 
+// EncodeConcatenatedHashed concatenates multiple hashes
+func EncodeConcatenatedHashed(hashes []*chainhash.Hash) (string, error) {
+	var buffer bytes.Buffer
+	for i := range hashes {
+		buffer.WriteString(hashes[i].String())
+	}
+	return buffer.String(), nil
```
Comment (chappjc) [inline dcrjson/parse.go]: Should there be an error returned at all?

Comment (alexlyp) [inline dcrjson/parse.go]: Typo?  Should be Hashes not Hashed?

Comment (chappjc) [inline dcrjson/parse.go]: yeah, typo

**Comment by chappjc** [comment]
Oh, I see.  This was all wrong.  Should have studied the Decode first.

**Comment by chappjc** [comment]
But https://github.com/decred/dcrrpcclient/pull/39 needs this.  Not sure I have time for this.  Was implementing this planned?

**Comment by alexlyp** [comment]
Yup @cjepson has actually just opened an issue and asked me to do a couple hours before you pushed PR :)
https://github.com/decred/dcrrpcclient/issues/38

**Comment by chappjc** [comment]
Haha.  I updated the dcrstakepool PR https://github.com/decred/dcrstakepool-private/pull/32 which is why I wanted the dcrrpcclient edit.

**Pull request closed without merging**